pkg/server/operands_handler.go

```diff
@@ -0,0 +1,73 @@
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/openshift/console/pkg/serverutils"
+	"github.com/operator-framework/kubectl-operator/pkg/action"
+	"k8s.io/klog"
+)
+
+type OperatorMeta struct {
+	Name      string
+	Namespace string
+}
+
+func operandsListHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
+		w.Header().Set("Allow", "GET")
+		serverutils.SendResponse(w, http.StatusMethodNotAllowed, serverutils.ApiError{Err: "Method unsupported, the only supported methods is GET"})
+		return
+	}
+	operatorMeta := parseOperatorNameAndNamespace(r.URL.Path)
+
+	cfg := &action.Configuration{}
+	err := cfg.Load()
```

Comment:
I'm not familiar with how this works. Does it discover the right configuration from the environment since it's running in a pod? Does this work for a development environment?

We'll want to support this for multicluster as well when we have the ACM integration.

Reply:
So from what I understand its getting the right configuration from the CLI, since its `kubectl-operator`  is a kubectl plugin. The `Load()` will use the oc/kubectl configuration context (namespace, ...) and also create needed clients. Since we are using it diretcly its best to call the `Load()` go get those clients, then set the namespace and then make the API call.

Reply:
> We'll want to support this for multicluster as well when we have the ACM integration.

in that case the kubectl-operator has to be installed on all the managed clusters in order to list the opernds.

Reply:
OK, this sounds like it will be an issue in 4.9 since we won't have any kubectl configuration for spoke clusters when we implement multi-cluster. We need to work with the OLM team to understand if we can initialize the client another way (API server URL + CA file + access token).

Is this using the console service account token to make the request?

Does this work in a console development environment?

Reply:
CC'ing @joelanford for more detailed answer. Thanks

Reply:
If `cfg.Load()` is run from a pod in a cluster and KUBECONFIG is unset, it _should_ load the in-cluster configuration from the mounted service account directory. If it doesn't, that's a bug that we would fix.

If you have more specific logic for where to pull the configuration from, the `action.Configuration` struct's fields are exported and can be manually configured as needed.

Reply:
@jhadvig Have you tested this with an in-cluster console? I don't see a `$HOME/.kube/config` or `$KUBECONFIG` set in the console pod, so I wouldn't expect it to work.

Reply:
@spadgett right, but `Load()` is also setting the clients for the `Configuration` object.

Reply:
> Does this work in a console development environment?

when running locally you have to have [installed the kubectl plugin ](https://github.com/operator-framework/kubectl-operator#install) and logged in into cluster via oc/kubectl

Reply:
> Is this using the console service account token to make the request?

I think it should, but would rather double check with @joelanford